### ISSUE TITLE
Support for pages

### ISSUE THREAD

**Issue by lincolnthree** [brool/wordpress-shuffle#3]
Just wondering :) I noticed your comment in the TODO items of the script. It would be great if Pages worked as well, but I don't see anything about this in the XMLRPC MetaWeblog or wordpress docs.

**Comment by brool** [comment]
Yeah, had to use the Wordpress API for this.  Added in commit ddaf0442a8699ffa328b5385ad26f8e8741b199d.

**Comment by lincolnthree** [comment]
Hmm. Actually - this is partly working. Pages with the same name (different URL) cannot be written

**Comment by brool** [comment]
You have two pages with the same slug line?  Or exact same title?  Would it be possible for me to see the metadata for the two files in question?

**Comment by lincolnthree** [comment]
Sure, no problem. Do you have a chat account? I'm
lincolnbaxter@gmail.comon gChat.

On Wed, Apr 25, 2012 at 6:31 PM, Tim Lopez <
reply@reply.github.com

> wrote:
> 
> You have two pages with the same slug line?  Or exact same title?  Would
> it be possible for me to see the metadata for the two files in question?
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/brool/wordpress-shuffle/issues/3#issuecomment-5345383

## 

Lincoln Baxter, III
http://ocpsoft.org
"Simpler is better."

**Comment by lincolnthree** [comment]
Where can I send you the dump?

**Comment by lincolnthree** [comment]
Here it is: https://github.com/lincolnthree/wordpress-shuffle/blob/dump/ocpsoft.org.tar.gz

**Comment by brool** [comment]
Thanks.  It looks like we're using the permalinks differently, I'll take a look at this.
